.github/workflows/tests.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         environment: [mindeps, "3.9", "3.10", "3.11", "3.12"]
-        label: [queue]
+        label: [default]
```

Comment:
This will make our test report look slightly uglier for the next 2 weeks; I don't think it's worth writing a migration layer for it.